README.md

```diff
@@ -1 +1,56 @@
 # MediaMonks - Frontend Coding Standards
+
+## Definition of Done Checklist
+
+#### Before Starting feature
+- [x] Read the ticket. If no ticket is present please ask the PM for one.
+- [x] Make sure the ticket is clear and actionable. If not, reassign the ticket to your project manager until the ticket is 100% clear.
+- [x] Create a feature branch (`feature/ticket-number-feature-name`)
+
+#### General Tasks
+- [x] Double check if feature is properly working on all browsers 
```

Comment:
Should we add some specificity here ? All browsers is a bit broad understanding.

- [x] Double check if feature is properly working on all browser specified in the browser matrix.